src/main/kotlin/pl/edu/agh/gem/external/persistence/MongoNotVerifiedUserRepository.kt

```diff
@@ -26,6 +28,14 @@ class MongoNotVerifiedUserRepository(
         mongo.remove(query, NotVerifiedUserEntity::class.java)
     }
 
+    override fun updateVerificationCode(id: String, newCode: String) {
```

Comment:
for future tip if you want you can use mongooperation.save() and then if id exists in DB it will update all other filds or if dont exists that id it will insert that